### ISSUE TITLE
Abmeldelink in Mails der Gruppen

### ISSUE THREAD

**Issue by NicoleRauch** [softwerkskammer/Agora#1321]
* [ ] Abmeldelink bereitstellen (GET-Request)
* [ ] Abmeldelink in Mails einfügen

* [ ] Idealerweise Link so gestalten, dass er schon anzeigt, mit welchem Provider man sich angemeldet hatte (also z. B. /groups/unsubscribe/:group/:nickname )

**Comment by NicoleRauch** [comment]
Scheint erforderlich zu sein:

https://www.deutsche-handwerks-zeitung.de/checkliste-so-duerfen-sie-per-e-mail-werben/150/3100/209583

**Comment by leider** [comment]
Keine Ahnung, warum das nicht über reply-to funktioniert. Wie jemand meinte: ezmlm kann das.

-> Eventuell Job für "Spezialist"

**Comment by NicoleRauch** [comment]
Warum WAS nicht über ezmlm funktioniert?

**Comment by leider** [comment]
unsubscribe - irgendwer meinte, das ginge nicht

**Comment by NicoleRauch** [comment]
Nein, das haben wir abgeklemmt (Einstellung in ezmlm), weil wir ja auch in der Agora Infos über das subscribe und unsubscribe mitführen (glaube ich) und deswegen über die Agora gehen wollen und nicht nur über ezmlm. Oder?

(Aber das ist kein reply-to, sondern eine Mail an eine andere Mailadresse.)

**Comment by NicoleRauch** [comment]
* "Wie können wir das OHNE Authentifizierung machen?" -> Soll nicht ohne Authentifizierung laufen.
* "dann könnten wir auch" -> klar! Aber mit irgendwas muss man mal anfangen, und ich würde gern mit dem dringendst benötigten anfangen. Dann können weitermachen.
* "auseinanderlaufen" -> Das wäre auch gut, das mal geradezuziehen.

**Comment by leider** [comment]
Das mit der Übersicht in der Liste und der zusätzliche Information ist schon so gut wie fertig. :)
